ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
+$icon-width: 24px;
+$icon-margin-right: 2rem;
+$question-padding: 1.5rem;
+
+.faq-block__icon {
+    font-size: 30px;
+    margin-right: $icon-margin-right;
+}
+
+.faq-block__question {
+    margin: 0;
+}
+
+.faq-block__text {
+    white-space: pre-wrap;
+}
+
+.mat-expansion-panel {
+    border-top: 2px solid $border-color;
+    font-family: Untitled-Sans-Regular;
+    font-size: 24px;
+
+    &:last-child {
+        border-bottom: 2px solid $border-color;
+    }
+
+    :hover, :focus {
+        background-color: rgba(0,0,0,.04);
+    }
+
+    .mat-icon,
+    .faq-block__question,
+    .faq-block__text {
+        background-color: transparent;
+    }
+}
+
+.mat-expansion-panel-header {
+    padding: 1.5rem;
+    transition: 0.3s;
+}
+
+:host ::ng-deep {
```

Comment:
why do we need it here?

Reply:
Not sure if I got your question.
the styles (paddings) in order to align a question text with a question title on the left.
'ng-deep' as to apply the styles into the child element.